editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -34,27 +34,43 @@
 #include "editor/editor_file_system.h"
 #include "editor/editor_node.h"
 #include "editor/editor_scale.h"
+#include "editor/filesystem_dock.h"
+
+#define CHECK_ADDON_INITIALIZED() \
+	ERR_FAIL_COND_MSG(!EditorVCSInterface::get_singleton(), "No VCS plugin is initialized. Select a Version Control Addon from Project menu.");
```

Comment:
It would be nice to turn all references to the godot-git-plugin e.g. from `ADDON` to `PLUGIN`, just so that it remains consistent throughout the editor and the plugin.

Reply:
After the macro name change, you will require a change to the error message here as well: "Select a Version Control addon" -> "Select a Version Control System plugin"